install_process.go

```diff
@@ -82,11 +86,11 @@ func onlineArgs(cachePath string) []string {
 	}
 }
 
-func offlineArgs(vendorDir string) []string {
+func offlineArgs(vendorDir string, requirementPath string) []string {
 	return []string{
 		"install",
 		"--requirement",
-		"requirements.txt",
+		requirementPath,
```

Comment:
Should I use `fmt.Sprintf`?

```suggestion
		fmt.Sprintf("--requirement=%s", requirementPath),
```